src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Constants;
+import core.basesyntax.service.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String fileName) {
+        String filePath = Constants.RESOURCE_PATH + fileName;
+        List<String> result = List.of();
+        try (Stream<String> lines = Files.lines(Paths.get(filePath))) {
+            result = lines.skip(1).collect(Collectors.toList());
+        } catch (IOException e) {
+            System.out.println("Can't find file on path: " + filePath);
+            ;
+        }

```

This violates checklist item #6: "Don't ignore exceptions in try-catch blocks". Printing the error to the console and continuing with an empty list is considered ignoring the exception. You should re-throw a `RuntimeException` with an informative message and the original exception object to properly signal that a failure has occurred.